hscontrol/db/db.go

```diff
@@ -360,10 +361,17 @@ func openDB(dbType, connectionAddr string, debug bool) (*gorm.DB, error) {
 		return db, err
 
 	case Postgres:
-		return gorm.Open(postgres.Open(connectionAddr), &gorm.Config{
+		db, err := gorm.Open(postgres.Open(connectionAddr), &gorm.Config{
 			DisableForeignKeyConstraintWhenMigrating: true,
 			Logger:                                   dbLogger,
 		})
+
+		sqlDB, _ := db.DB()
+		sqlDB.SetMaxOpenConns(10)
+		sqlDB.SetMaxIdleConns(10)
+		sqlDB.SetConnMaxIdleTime(time.Hour)
```

Comment:
Should we make this user configurable?

Reply:
> Should we make this user configurable?

Thanks, that's a good idea. I'll create another commit with the changes required to make these fields configurable.

Reply:
@kradalby I've pushed the commit. Please take a look. 🙇🏼‍♂️ 

Thanks.